packages/odyssey-react/src/components/Tab/index.tsx

```diff
@@ -65,6 +65,7 @@ export type PropsTabsPanel = {
   selected?: boolean;
 };
 export interface PropsTabsList
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
This file has a larger effort required to fix all the type problems (static implementations are not typed etc) so I'm punting to get this merged for now - we'll revisit `Tab` soon to fix up the types